layouts/shortcodes/proposal-tables.html

```diff
@@ -60,6 +60,7 @@
             {{ end }}
             {{ $docs_links := delimit $docs_links_list ", " }}
 
+            {{ $authors_list := apply .authors "htmlEscape" "." }}
             {{ $authors_list := apply .authors "printf" "<a href=\"https://github.com/%s\">@%s</a>" "." "." }}
```

Comment:
I don't really understand this. Isn't the new value immediately overrwritten?

Reply:
Oops, it's using the wrong variable here. The `htmlEscape` is a only precaution anyway in case there is a special HTML character in the username of an author, but I'm not even sure GitHub would allow that.

Reply:
I remember now why I did that, even if GitHub doesn't allow special characters in the username, it's because the data from the MSC description is not validated.

Reply:
Ad now I realized that for the URL, we should not use `htmlEscape` but rather a function that converts to a URL-compatible format. Hugo has `urlize` but it also transforms the string instead of just converting special characters.

Maybe we should consider sanitizing to be the job of the script that fetches the proposals?

Reply:
I removed the call to `htmlEscape`, maybe it can be done separately, to see if we want to sanitize the input of the proposals.